docs/src/analysis/sensitivity.md

```diff
@@ -383,3 +385,75 @@ end
 res2 = ForwardDiff.gradient(G,[1.5,1.0,3.0])
 res3 = Calculus.gradient(G,[1.5,1.0,3.0])
 ```
+
+## Global Sensitivity Analysis
+
+Global Sensitivity Analysis methods are used to quantify the uncertainity in output of a model w.r.t.
+the parameters, their individual contributions or the contribution of their interactions. The type of GSA method
+to use depends on the interest of the user, below we describe the methods available in the suite at the moment (some more are already in development)
+and explain what is the output of each of the methods and what it represents. 
+
+### Morris Method
+
+The Morris method also known as Morris’s OAT method where OAT stands for One At a Time can be described in the following steps:
+
+We calculate local sensitivity measures known as “elementary effects”, which are calculated by measuring the perturbation in the output of the model on changing one parameter. 
+
+$$ EE_i = \frac{f(x_1,x_2,..x_i+ \Delta,..x_k) - y}{\Delta} $$
+
+These are evaluated at various points in the input chosen such that a wide “spread” of the parameter space is explored and considered in the analysis, to provide an approximate global importance measure.
+The mean and variance of these elementary effects is computed.
+A high value of the mean implies that a parameter is important, a high variance implies that its effects are non-linear or the result of interactions with other inputs. This method fails to provide a proper quantification of the separate contribution from the interaction and the contribution of the parameters individually. 
+
+`morris_effects = morris_sensitivity(f,param_range,param_steps;relative_scale=false,kwargs...)`
+
+`morris_effects = morris_sensitivity(prob::DiffEqBase.DEProblem,alg,t,param_range,param_steps;kwargs...)`
+
+Here, `f` is just the model (as a julia function or a `DEProblem`) you want to run the analysis on, `param_range` requires an array of your range for the parameters, `param_steps` decides the value of \Delta in the equation above and `relative_scale`, the above equation takes the assumption that the parameters lie in the range `[0,1]` but as this is not always the case scaling is used to get more informative, scaled effects.
```

Comment:
These do not adequately describe what the inputs have to be. "requires an array of your range of parameters", so it's an array of numbers? These needs more detail